cea/optimization/slave/least_cost.py

```diff
@@ -91,11 +91,16 @@ def least_cost_main(locator, master_to_slave_vars, solar_features, gv):
     # Import Temperatures from Network Summary:
 
     os.chdir(locator.get_optimization_network_results_folder())
```

Comment:
avoid the use of these local paths. do instead VARIABLE = locator.get_optimization_network_results_folder() if needed